core-ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -267,9 +267,35 @@ fun NewsResourceShortDescription(
 
 @Composable
 fun NewsResourceTopics(
-    newsResource: NewsResource
+    topics: List<Topic>,
+    modifier: Modifier = Modifier
 ) {
-    TODO()
+    LazyRow(
+        modifier = modifier,
+        horizontalArrangement = Arrangement.spacedBy(4.dp)
+    ) {
+        items(items = topics, key = { item -> item.id }) { topic ->
+            NiaTopicChip(
+                enabled = true, // ToDo: Chip should be disabled if user is not following
+                onClick = { }, // ToDo: Handle topic chip interaction
+            ) {
+                Text(uppercaseFormat(topic.name))
+            }
+        }
+    }
+}
+
+@Composable
+private fun uppercaseFormat(string: String): String {
+    val locale = ConfigurationCompat.getLocales(LocalConfiguration.current).get(0)
+
+    val uppercaseFormatted = if (locale != null) {
+        string.uppercase(locale)
+    } else {
+        string.uppercase()
+    }
+
+    return uppercaseFormatted
```

Comment:
I feel like there should be an easier fool-proof way to uppercase a String. Asked about it in chat.

Reply:
removed in favor of `uppercase(Locale.getDefault())` based on chat. seems to be what most compose samples are using too https://github.com/android/compose-samples/search?q=uppercase